src/state/swap/hooks.tsx

```diff
@@ -558,7 +558,8 @@ export function usePoolAddress(
 
     // no address; set default
     if (poolAddress == '' || poolAddress == undefined || poolAddress == null) {
-      poolAddress = (chainId && KROM[chainId].address) || '0x3af33bEF05C2dCb3C7288b77fe1C8d2AeBA4d789'
+      // poolAddress = (chainId && KROM[chainId].address) || '0x3af33bEF05C2dCb3C7288b77fe1C8d2AeBA4d789'
```

Comment:
Please do not leave comments unless they are need to describe some method or complex logic.